Rmarkdown/Multiple_Ascending_Dose_PD_time_to_event.Rmd

```diff
@@ -1,73 +1,97 @@
 ---
 title: "PD, Dose-Response - Time to Event"
+author: "Andy Stein, Anwesha Chaudhury, Alison Margolskee"
 output:
   html_document:
-    toc: true
-    toc_float: true
     code_folding: hide
+    toc: yes
```

Comment:
Changing the "output" settings have negatively impacted the compilation of the website in the past. Please change back to the below. Also removing the pdf_document lines

output:
  html_document:
    toc: true
    toc_float: true
    code_folding: hide

Reply:
done.